subsys/bluetooth/host/buf.c

```diff
@@ -18,20 +18,12 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_buf, CONFIG_BT_LOG_LEVEL);
 
-#if defined(CONFIG_BT_CONN)
-#if defined(CONFIG_BT_ISO)
-#define MAX_EVENT_COUNT CONFIG_BT_MAX_CONN + CONFIG_BT_ISO_MAX_CHAN
-#else
-#define MAX_EVENT_COUNT CONFIG_BT_MAX_CONN
-#endif /* CONFIG_BT_ISO */
-#elif defined(CONFIG_BT_ISO)
-#define MAX_EVENT_COUNT CONFIG_BT_ISO_MAX_CHAN
-#endif /* CONFIG_BT_CONN */
-
-#if defined(CONFIG_BT_CONN) || defined(CONFIG_BT_ISO)
-#define NUM_COMLETE_EVENT_SIZE BT_BUF_EVT_SIZE(                        \
-	sizeof(struct bt_hci_cp_host_num_completed_packets) +          \
-	MAX_EVENT_COUNT * sizeof(struct bt_hci_handle_count))
+/* Events have a length field of 1 byte. This size fits all events.
+ *
+ * It's true that we don't put all kinds of events there (yet). However, the
+ * command complete event has an arbitrary payload, depending on opcode.
+ */
+#define SYNC_EVT_SIZE (BT_BUF_RESERVE + BT_HCI_EVT_HDR_SIZE + 255)
```

Comment:
Consider adding `MAX` somewhere in the name so that it's not confused with something like `CONFIG_BT_BUF_EVT_RX_SIZE` that is relative